src/Rule.php

```diff
@@ -3483,7 +3476,7 @@ public function initRules(bool $reset = true, $with_plugins = true, $check = tru
 
             $rule = new static();
 
-            if ($check === true && $rule->getFromDBByCrit(['uuid' => (string)$rulexml->uuid])) {
+            if ($rule->getFromDBByCrit(['uuid' => (string)$rulexml->uuid])) {
```

Comment:
Previously, check was set to false when reset was done. It has no sense to check if all has been deleted already.